appcircle_audit_permission_changes/1.0.0/component.yaml

```diff
@@ -0,0 +1,22 @@
+platform: iOS
+buildPlatform:
+displayName: Audit Permission Changes
+description: "Permission Differance Checker"
+webUrl: https://github.com/appcircleio/appcircle-ios-permission-check-component
```

Comment:
**main.rb**:

- [ ] 1. line = `open3` package not using in the code
- [ ] 7. line = `English` package not using in the code
- [ ] 8. line = `fileutils ` package not using in the code
- [ ] 9. line = `os ` package not using in the code
- [ ] 16. line => Why you use  `start_with?('$') ? ENV[value[1..-1]] `
- [ ] 31. line => what is different `get_env()` function from `get_env_variable()`
- [ ] 35, 36, 37, 39, 46, 47. line => If there is no relevant env variable in these lines, a Missing error should be returned. However, the "get_env_variable()" function does not return this error.
- [ ] 57, 58. line => variables here are defined in more than one place.
- [ ] 93. line => update_target() function didn't use in code.
- [ ] 256, 499. line => run_command() duplicated
- [ ] 261, 514. line => abort_with0() duplicated
- [ ] 280,281,282,283 line => Log have not any desc. 
- [ ] ...
There are many similar complexity errors after this line. Since these errors make it difficult for me to review the code, it would be more logical to review them again after fixing them.